engine/include/cubos/engine/renderer/renderer.hpp

```diff
@@ -93,11 +93,13 @@ namespace cubos::engine
         /// @param viewport Camera viewport.
         /// @param camera Camera to use.
         /// @param frame Frame to draw.
+        /// @param pickingBuffer Screen picking framebuffer.
+        /// @param enableScreenPicking Whether to draw to the screen picking buffer.
         /// @param usePostProcessing Whether to use post processing.
         /// @param target Target framebuffer to draw to.
         void render(const glm::mat4& view, const Viewport& viewport, const engine::Camera& camera,
-                    const RendererFrame& frame, bool usePostProcessing = true,
-                    const core::gl::Framebuffer& target = nullptr);
+                    const RendererFrame& frame, const core::gl::Framebuffer& pickingBuffer, bool enableScreenPicking,
```

Comment:
Instead of passing a boolean `enableScreenPicking`, you can set `pickingBuffer` to `nullptr` to disable it, as we do with `target`.